Dockerfile

```
@@ -0,0 +1,37 @@
+ARG BUILDER_IMAGE
+ARG AWS_CLI_VERSION
+FROM ${BUILDER_IMAGE} AS builder
+
+WORKDIR /usr/src/app
+
+RUN python3 -m venv /venv
+ENV PATH="/venv/bin:$PATH"
+
+RUN pip install --upgrade pip
+#RUN apk add cargo
+
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt 
+
+FROM ${BUILDER_IMAGE} AS service
+
+# Install the aws cli v2, which is now default in alpine
+ARG AWS_CLI_VERSION
+RUN apk update && apk add aws-cli==${AWS_CLI_VERSION} bash zip curl
+
+#ENV PATH=/usr/bin:/venv/bin:/root/app/site-packages/bin:$PATH
+ENV PATH=/usr/bin:/venv/bin:$PATH
+WORKDIR /app/site-packages
+COPY --from=builder /venv /venv
+
+# Add awslocal to site-packages/bin, with priority to awsv2 bin
+# As awscli is added for the bindings, v1 will be also added
+COPY bin/awslocal-docker /venv/bin/awslocal
```

I don't think we need the copy of the file
```suggestion
COPY bin/awslocal /venv/bin/awslocal
```

---

Yeah, the whole problem here is that the image has python 3.13 installed with the command venv creation... I will remove that step and see if it's possible to keep a single installation and, as a result, decrease the image size.